Data/Primitive/Addr.hs

```diff
@@ -22,11 +22,16 @@ module Data.Primitive.Addr (
   indexOffAddr, readOffAddr, writeOffAddr,
 
   -- * Block operations
-  copyAddr, moveAddr, setAddr
+  copyAddr,
+#if __GLASGOW_HASKELL__ >= 708
+  copyAddrToByteArray,
+#endif
+  moveAddr, setAddr
 ) where
 
 import Control.Monad.Primitive
 import Data.Primitive.Types
+import Data.Primitive.ByteArray
```

Comment:
Guard this import with `#if __GLASGOW_HASKELL__ >= 708` too. Otherwise, it'll be redundant on older GHCs.

Reply:
done